gulp-tasks/wfContributors.js

```diff
@@ -69,6 +98,7 @@ function buildAll() {
   var contributors = getContributors();
   buildIncludes(contributors);
   buildIndex(contributors);
```

Comment:
Would you please move the `contributors.md` file from `resources` into the `contributors` folder, and name it `index.md`. Then, please check that any URLs are updated.

Reply:
Done